modules/android/pages/_partials/pn-issues-list.adoc

```diff
@@ -1,5 +1,57 @@
 = Release Note issues for  Android
 
+// tag::issues-3-0-2[]
+
+== Fixed
+
+// tag::Fixed-3-0-2[]
+
+* https://issues.couchbase.com/browse/CBL-3301[CBL-3301] -- https://issues.couchbase.com/browse/CBL-3301[FullTextIndex.setLanguage() should accept null parameter]
+
+* https://issues.couchbase.com/browse/CBL-3299[CBL-3299] -- https://issues.couchbase.com/browse/CBL-3299[Fix NPE in finalizer]
+
+* https://issues.couchbase.com/browse/CBL-3136[CBL-3136] -- https://issues.couchbase.com/browse/CBL-3136[Malformed hostname can cause a crash]
+
+* https://issues.couchbase.com/browse/CBL-3000[CBL-3000] -- https://issues.couchbase.com/browse/CBL-3000[c4DbChanges in AbstractDatabase.postDatabaseChanged may contain nulls]
+
+* https://issues.couchbase.com/browse/CBL-2998[CBL-2998] -- https://issues.couchbase.com/browse/CBL-2998[Replicator stopped with WebSocket 1008 / SocketTimeoutException when disable Wi-Fi]
+
+* https://issues.couchbase.com/browse/CBL-2996[CBL-2996] -- https://issues.couchbase.com/browse/CBL-2996[Several bad tests in ReplicatorLocal2LocalTest suite]
+
+* https://issues.couchbase.com/browse/CBL-2995[CBL-2995] -- https://issues.couchbase.com/browse/CBL-2995[Cookies set in Headers are replaced with Session Authenticator's cookies]
+
+* https://issues.couchbase.com/browse/CBL-2962[CBL-2962] -- https://issues.couchbase.com/browse/CBL-2962[URLEndpointListenerTest fixes]
+
+
+// end::Fixed-3-0-2[]
+
+
+== Enhancements
+
+// tag::Enhancements-3-0-2[]
+
+* https://issues.couchbase.com/browse/CBL-3361[CBL-3361] -- https://issues.couchbase.com/browse/CBL-3361[As of Kotlin 1.6, Coroutines are no longer experimental]
+
+* https://issues.couchbase.com/browse/CBL-3274[CBL-3274] -- https://issues.couchbase.com/browse/CBL-3274[Expose the IndexConfiguration class]
+
+* https://issues.couchbase.com/browse/CBL-3084[CBL-3084] -- https://issues.couchbase.com/browse/CBL-3084[Make SSLException a recoverable error in AbstractCBLWebSocket]
+
+* https://issues.couchbase.com/browse/CBL-2999[CBL-2999] -- https://issues.couchbase.com/browse/CBL-2999[Implement enhanced pinned server certificate feature]
+
+// end::Enhancements-3-0-2[]
+
+== KI
```

Comment:
Took me a second to understand what this was, would it not be better to say it explicitly: `Known Issues`.
Or is it something else?

Reply:
Yes, I'm going to do a quick document to explain how this works.

The `== KI` is just a placeholder to show where you fill in the Known Issues; the heading doesn't actually show up when the pages are compiled.